src/engine/v3/V3Interpreter.v3

```diff
@@ -1139,6 +1145,28 @@ component V3Interpreter {
 		}
 		push(Value.V128(low, high));
 	}
+	def doReplaceLane<T, S>(log2_size: u3, view: S -> T, unbox: Value -> S) {
```

Comment:
You could also pass one of `popi`, `popf`, etc, here.

Reply:
Ah yes that's a miss. Updating it.